lib/ui/widgets/password_text_field.dart

```diff
@@ -32,13 +34,13 @@ class _PasswordFormFieldState extends State<PasswordFormField> {
                     color: Colors.white, width: borderRadiusOnFocus)),
             focusedErrorBorder: OutlineInputBorder(
                 borderSide: BorderSide(
-                    color: Colors.amberAccent, width: borderRadiusOnFocus)),
+                    color: getErrorTextColor(), width: borderRadiusOnFocus)),
             errorBorder: OutlineInputBorder(
                 borderSide: BorderSide(
-                    color: Colors.amberAccent, width: borderRadiusOnFocus)),
+                    color: getErrorTextColor(), width: borderRadiusOnFocus)),
             labelText: AppIntl.of(context).login_prompt_password,
             labelStyle: const TextStyle(color: Colors.white54),
-            errorStyle: const TextStyle(color: Colors.amberAccent),
+            errorStyle: TextStyle(color: getErrorTextColor()),
```

Comment:
```suggestion
            errorStyle: TextStyle(color: errorTextColor),
```

If you apply the change below 👇